packages/constants/src/assets.ts

```diff
@@ -11,6 +11,11 @@ export const STAKING_TOKEN_METADATA = localAssets.find(
   metadata => metadata.display === STAKING_TOKEN,
 )!;
 
+export const NUMERAIRE_TOKEN = 'test_usd';
+export const NUMERAIRE_TOKEN_ID = localAssets.find(
+  metadata => metadata.display === NUMERAIRE_TOKEN,
+)!.penumbraAssetId!;
+
```

Comment:
In the future we can easily change the `NUMERAIRE_TOKEN` if necessary

Reply:
See `apps/extension/.env`. We should try to use env variables for this given mainnet/testnet will have a different numeraire. Looks like we started this already with `USDC_ASSET_ID`. We should update or replace that.